src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Application {
+    private static final int NUMBER_OF_FIGURES = 6;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[NUMBER_OF_FIGURES];
+
+        for (int i = 0; i < NUMBER_OF_FIGURES; i++) {
+            figures[i] = (i < (figures.length / 2)) ? figureSupplier.getRandomFigure()
```

Comment:
Usually ternary operator reduces code readability 
So it's better to use `if - else` in such case